jwst/flatfield/flat_field.py

```diff
@@ -74,8 +74,8 @@ def do_correction(input_model,
     # NIRSpec spectrographic data are processed differently from other
     # types of data (including NIRSpec imaging).  The test on flat is
     # needed because NIRSpec imaging data are processed by do_flat_field().
-    if (input_model.meta.exposure.type in NIRSPEC_SPECTRAL_EXPOSURES) and \
-            (input_model.meta.instrument.lamp_mode != 'IMAGE'):
+    if ((input_model.meta.exposure.type in NIRSPEC_SPECTRAL_EXPOSURES) and
+            (input_model.meta.instrument.lamp_mode != 'IMAGE')):
```

Comment:
```suggestion
    if (input_model.meta.exposure.type in NIRSPEC_SPECTRAL_EXPOSURES and
        input_model.meta.instrument.lamp_mode != 'IMAGE'):
```

Reply:
I prefer the indent on the second line because it gives better visual separation from the code being executed under the condition, but I'm fine with your suggested formatting.  Whatever is decided, though, I think it should be standard.